### ISSUE TITLE
Does the WCAG 2.2 draft pass 1.3.2 and 2.4.3?

### ISSUE THREAD

**Issue by dd8** [w3c/wcag#3106]
The `<nav id="toc">` containing the Table of Contents is in the DOM between the 'Status of this Document' and 'Introduction' sections. CSS positioning moves it to a sidebar on desktop sized displays (but on mobile it's rendered in DOM order).

https://www.w3.org/TR/WCAG22/

That might be a failure of the test procedure in https://www.w3.org/TR/WCAG20-TECHS/F1.html for desktop devices. 

There's also a potential 2.4.3 Focus Order issue tabbing through the document on desktop. Tabbing moves down the document till you get to the last link in the 'Status of this Document' section then jumps to the first entry in the 'Table of Contents' in the sidebar. It eventually returns to the 'Introduction' section in the main document after the last link in the ToC.

**Comment by patrickhlauke** [comment]
i'd say no, not a failure. the *meaning* is not affected by the positioning / change of positioning (compare to F1 where the order/hierarchy of things is predicated purely on the visual presentation, and then meaning/order/grouping of things changes when CSS isn't applied or when you read/navigate the content sequentially in DOM order as an SR user would, for instance).

also, for 2.4.3, this is not a failure in my view as focus doesn't jump illogically/haphazardly ... it is still logical in how it moves around the page. yes, it doesn't necessarily follow the "visual" order, but that's not required by 2.4.3 (an aspect i've tried to cover more in https://github.com/w3c/wcag/pull/1643)

**Comment by dd8** [comment]
1.3.3 - agreed
2.4.3 - is less clear to me - https://github.com/w3c/wcag/pull/1643 says "and the hierarchy and relationship of content implied by the visual presentation is preserved". Arguably the document hierarchy in the right hand column is not been followed because focus jumps out the document hierarchy on the right to the ToC on the left before returning. Although later the PR says "Focus order does not need to follow visual presentation or layout". Seems there's scope for different auditors to come to different conclusions on this.

Another issue with the ToC is there's no easy way for keyboard users to skip it when tabbing, unless the user figures out that the third link in the ToC (Introduction) brings them back to the part of the document after the ToC (but that's not very obvious). Not an issue for screen reader users who can use use next heading key.
